sites/feedback/reports/acr-2025-06.json

```diff
@@ -124,28 +267,60 @@
       "earl:test": [ "WCAG21:status-messages" ],
       "acr:severity": "acr:notRelevant",
       "acr:relevancy": "wbv:acr#comments"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Improve WET-BOEW Postback plugin",
+      "wf:goalDescription": "The WET-BOEW Postback plugin should be improved so that the confirmation and error messages are announced by all popular screen-readers. The current solution only displays the message which is not announced by JAWS or NVDA because no text change happened.",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:status-messages",
+      "earl:test": [ "WCAG21:status-messages" ],
+      "acr:severity": "acr:moderate",
+      "acr:relevancy": "wbv:acr#usability"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Add focus to text box on click of No button",
+      "wf:goalDescription": "When users click the No button, give the focus to the newly displayed text box so that users don't have to manually move to it.",
+      "dct:created": "2025-06-11",
+      "acr:requirement": "WCAG21:focus-order",
+      "earl:test": [ "WCAG21:focus-order" ],
+      "acr:severity": "acr:moderate",
+      "acr:relevancy": "wbv:acr#usability"
+    },
+    {
+      "@type": "wf:Task",
+      "dct:title": "Enhance redacted text screen-reader alternative",
+      "wf:goalDescription": "Enhance the screen-reader text of the redacted text so that it reads \"personal information redacted\" instead of just \"redacted\".",
```

Comment:
Update this task to indicate to perform the research first, building the evidence, prior to perform the propose change.
This need to be validated with some user group.